src/lib/chips/_chips-theme.scss

```diff
@@ -7,12 +7,17 @@
   $warn: map-get($theme, warn);
   $background: map-get($theme, background);
 
+  // TODO: Should this be in chips.scss since it is independent of theme?
   .md-chip {
     background-color: #e0e0e0;
     color: rgba(0, 0, 0, 0.87);
   }
 
   .md-chip.selected {
+    // TODO: Based on spec, this should be #808080, but we can only use md-contrast with a palette
+    background-color: md-color($md-grey, 600);
+    color: md-contrast($md-grey, 600);
```

Comment:
I can use a custom background color, but is there a good way to do the `md-contrast()` with a standard color instead of the palette color? If not, I guess I can manually calculate it based on the spec.